core/Services/CmfPackageController.cs

```diff
@@ -1126,6 +1126,11 @@ string getUniquePackageAlias(string packageName)
                                 new JProperty("mandatoryDependencies", mandatoryDependecies),
                                 new JProperty("conditionalDependencies", conditionalDependencies));
 
+        if (addManifestVersion)
```

Comment:
Should we verify if the jso already has a manifest version set before setting to 1.0?

Reply:
We are forcing this value at the time of publishing as intended to prevent serialization/deserialization of this property.
Check [this thread](https://github.com/criticalmanufacturing/cli/pull/538#discussion_r2194680470).

Reply:
I think Pinho's point is that we could be publishing a package.json that already has the `deployment.manifestVersion` set.

Right now, packages generated using the cmf cli do not have one, but we could be publishing a package created in some other way (manually, for example). It would be nice and it would't hurt, to check if the `manifestVersion` already exists inside the `deployment` property. 😁